src/test/java/org/sagebionetworks/bridge/services/StudyServiceTest.java

```diff
@@ -151,6 +158,7 @@ public void createStudy() {
         study.setModifiedOn(timestamp);
 
         when(studyDao.createStudy(any())).thenReturn(VERSION_HOLDER);
+        when(organizationDao.getOrganization(TEST_APP_ID, TEST_ORG_ID)).thenReturn(Optional.of(Organization.create()));
```

Comment:
This line doesn't appear to do anything. Did we forget a code change somewhere?

Reply:
Yes, I had toyed with forcing an orgId to be provided when creating a study, so it would start with one sponsor organization. But it seemed unnecessary since it's easy to associate/disassociate things after the fact if you need to.